Classes/Request/AbstractRequest.php

```diff
@@ -148,8 +148,7 @@ public function __sleep()
 
     /**
      * Returns the mandant
-     *
-     * @ToDo: Currently only one mandant can be configured. Maybe we will allow multiple mandants with a configuration record or pageTSconfig
+     * ToDo:Currently only one mandant can be configured. Maybe we will allow multiple mandants with a configuration record or pageTSconfig
```

Comment:
One space after : is missing :trollface:. I´ll approve it anyway :)

Reply:
no no...I'll correct that. Thank you